data/data/gcp/control/main.tf

```diff
@@ -0,0 +1,40 @@
+resource "google_compute_instance" "control" {
+  name         = "${var.cluster_id}-control-${count.index}"
+  count        = var.instance_count
```

Comment:
usually count is first field and better to separate from the rest with a newline...